main.py

```diff
@@ -0,0 +1,55 @@
+"""
+You need to create a github repo (public), plan project structure, use MIT license. 
```

Comment:
@MariaMaximova, could you please add a text indicating that this is a task? And maybe we can move the task to a README.md as there migbe be more files in the project?

Reply:
Possibly, the task might help to create a documentation to this tool.